### PR TITLE
Troubleshooting for misplaced/misspelled fields

### PR DESCRIPTION
**Pull request by erictune**
Inspired to write this while helping user with issue #12278.
For v1.0.2 this is necessary advice.
So, this should get cherry-picked to v1.0.
This may be fixed by #11914.  Therefore:
- suggest @nikhiljindal confirm my assumption that #11914 not going to be cherrypicked.
- suggest @nikhiljindal as reviewer
- suggest rewording docs once this is cherrypicked to v1.0.

### CODE REVIEW

**Comment by erictune** [comment]
Should be cherry-picked to v1.0.3.  Only a docs change.

**Comment by nikhiljindal** [comment]
Yes. We wont cherrypick 11914.
Doc change LGTM.
Thanks!

**Comment by wojtek-t** [comment]
Closed & reopened to trigger Shippable.

**Comment by wojtek-t** [comment]
It seems that Shippable failure is a real one/
@erictune can you please rebase it (this will hopefully help)
